src/utils.cpp

```diff
@@ -195,6 +196,104 @@ bool is_lit(const Term & l, const Sort & boolsort)
   return false;
 }
 
+
+
+std::string cnf_to_dimacs(Term cnf){
```

Comment:
Let's have this function take a second argument, similar to this function:
https://github.com/makaimann/smt-switch/blob/ed05b58d109642f440e6311c632131cdeb20991a/include/printing_solver.h#L45

instead of returning a string.

The string can be then obtained from the stream for testing.